### DISCUSSION TITLE
Which version of Astro does your template support?

### DISCUSSION THREAD

**Discussion by vasilytray** [mearashadowfax/ScrewFast#288] [Q&A] [answered]
Good afternoon. I have the June version of your template. I tried to upgrade to the latest version of Astro and components.  There are a lot of errors after the update. Please tell me which version of Astro is the latest version of your template running on?

**Comment by mearashadowfax** [comment, marked as answer]
Hi @vasilytray! The version of Astro is mentioned in the package.json file, and the latest is `4.16.17`. If you're upgrading to Astro v5, you may need to convert any existing content collections to the new Content Layer API and review the breaking changes listed in Astro's upgrade guide: https://docs.astro.build/en/guides/upgrade-to/v5/

**Reply by vasilytray** [reply]
OK, @mearashadowfax thanks for the reply 👍
